src/SFA.DAS.ASK.Web/Views/DeliveryPartner/PlanningMeetings/LogContact.cshtml

```diff
@@ -0,0 +1,99 @@
+﻿@model SFA.DAS.ASK.Web.ViewModels.DeliveryPartner.PlanningMeetings.LogContactViewModel
+@{
+    ViewData["Title"] = "Log Contact";
+}
+
+
+<main class="govuk-main-wrapper" id="main-content" role="main">
+    <div class="govuk-grid-row">
+        
+        <div class="govuk-grid-column-two-thirds">
+
+            <partial name="_ErrorSummary" />
+
+            <h1 class="govuk-heading-xl">
+                Log Contact
+                <span class="govuk-caption-xl"> @Model.EstablishmentName  </span>
+            </h1>
+
+            @if (Model.Status != SFA.DAS.ASK.Data.Entities.Status.ContactConfirmed)
```

Comment:
If the user shouldn't be getting to this page based on the status of the SupportRequest (which is correct), then the check for this should be in the controller really.  And should be switch around so that this page will only show *if the status is this* rather than if the status is not this.  If you see my meaning.